packages/frontend/src/pages/Search/SearchResult.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from '@mui/material';
+import { useSearchParams } from 'react-router-dom';
+import { Tags } from '../../components/Tags';
+import { ContainerWrapper } from '../Landing/index.styles';
+
+// TODO: Use real data fetched from Backend (currently use mock data)
+import mockData from './mockData.json';
```

Comment:
For future: Using env variable (`ENV=local` or such) to use mockData locally?

Reply:
Good question. Should also dev env use mockData if we deploy (ofc for temporary use to display for customer)?

Reply:
In dev we have the real API available, so we should use that. Otherwise we wouldn't be able to test API calls without going to prod 😅